libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/NavigationCamera.kt

```diff
@@ -422,15 +433,14 @@ class NavigationCamera(
         progressState: NavigationCameraState,
         finalState: NavigationCameraState,
         frameTransitionOptions: NavigationCameraTransitionOptions,
-        externalListener: Animator.AnimatorListener?
+        transitionEndListener: TransitionEndListener?,
```

Comment:
avoid set implicitly `runningTransitionEndListeners` and add any of the listeners to `runningTransitionEndListeners` that have already been used